tests/integration/suite/test_notifier.py

```diff
@@ -0,0 +1,20 @@
+from .common import random_str
+
+
+def test_notifier_update(admin_mc, remove_resource):
```

Comment:
The test name is too general concerning what's done in it. It would be better to change to test_notifier_wechat or test_notifier_wechat_apiurl